### DISCUSSION TITLE
【委托】谁有这个姐姐的资料？

### DISCUSSION THREAD

**Discussion by Luli-Chen** [share121/inter-knot#57] [其他]
![屏幕截图 2024-08-09 142102](https://github.com/user-attachments/assets/98159468-b8b6-4245-9409-958c3fcac1f1)
听说她是“山狮”的干部？有没有老哥能去探探？

**Comment by MouxiuSama** [comment]
探不了一点，会噶的，兄弟。女人如玫瑰，美丽的同时也带刺，别让小头控制大头啊

**Comment by Luli-Chen** [comment]
报酬丰厚啊各位

**Comment by Luli-Chen** [comment]
人就活这么一次啊，哥们会一直等的

**Comment by DSSM1031** [comment]
不知道，只知道她的皮鼓很大很肉，她小弟享福了

**Comment by Luli-Chen** [comment]
我有当小弟的机会吗

**Comment by 132641515Zzy** [comment]
不到啊，导说有学分我就来了

**Comment by koyki233** [comment]
不行，山师那边的委托给的太多了

**Comment by Luli-Chen** [comment]
> [❤点我看邦布可爱录像❤物美价廉❤]

TD.

**Comment by wushanef** [comment]
这个委托我接了

**Comment by Luli-Chen** [comment]
> 这个委托我接了

兄弟私聊

**Comment by Elysia041** [comment]
看起来是个美丽的女孩子呢！
这个委托我接了～ ♪

**Comment by wugan-bot156** [comment]
委托我接下了。这位你吃不消的，还是我来罢。*^____^*

**Comment by WLMRMZ** [comment]
这个委托，我会找其他绳匠的！

**Comment by GanlinSama** [comment]
wok，是绳匠形态的爱莉（飞扑）
